core/lib/refinery/cli.rb

```diff
@@ -1,7 +1,7 @@
 require 'thor'
 
 module Refinery
-  class CLI < Thor
```

Comment:
perhaps it should live inside `Core` too?

Reply:
I think we could leave it here until we rewrite it